Sources/ConsoleKit/Command/Async/AnyAsyncCommand.swift

```diff
@@ -0,0 +1,28 @@
+/// A type-erased `Command`.
+public protocol AnyAsyncCommand {
```

Comment:
This will need wrapping in compiler directives so the library still compiles in 5.2-5.4 and then anything that touches this